infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationStreamNameTableManager.java

```diff
@@ -70,6 +73,21 @@ public class LogReplicationStreamNameTableManager {
                     TableRegistry.PROTOBUF_DESCRIPTOR_TABLE_NAME))
     ));
 
+    @Getter
+    private static String currentVersion;
+
+    private Table<TableInfo, Namespace, CommonTypes.Uuid> streamsNameTable;
+
+    private Table<VersionString, Version, CommonTypes.Uuid> pluginVersionTable;
+
+    @VisibleForTesting
```

Comment:
I suppose the current tests do not need this?

Reply:
Removed, thanks!